AzureMonitorAgent/agent.py

```diff
@@ -99,8 +99,8 @@ def __str__(self):
 
 # Global Variables
 PackagesDirectory = 'packages'
-# TO BE CHANGED WITH EACH NEW RELEASE IF THE BUNDLE VERSION CHANGES
-# TODO: Installer should automatically figure this out from the folder instead of requiring this update
+# This changes during release if there are changes to mdsd. The release pipeline
```

Comment:
We'd need to change those to another package soon with AMA 2.0 repackaging, do we need to plan some back-compat here or we can simply have a separate 2.0 extension that only knows of 2.0 packages?

Reply:
Right now, it's split per-branch. See 'ama-2.0-preview' branch: https://github.com/Azure/azure-linux-extensions/blob/release/ama-2.0.0-preview/AzureMonitorAgent/agent.py